tapdb/universe_stats.go

```diff
@@ -728,7 +730,9 @@ func (u *UniverseStats) QueryAssetStatsPerDay(ctx context.Context,
 	}
 
 	// We have a fresh result, so we'll cache it now.
-	_, _ = u.assetEventsCache.Put(query, results)
+	if _, err := u.assetEventsCache.Put(query, results); err != nil {
+		log.Errorf("unable to query events cache: %v", err)
```

Comment:
I don't think the error message is correct here. The `Put` method call is not trying to query the events cache. Perhaps the error message should be:

```
log.Errorf("Unable to populate assets events cache: %v", err)
```